src/borg/remote.py

```diff
@@ -137,10 +137,11 @@ class RemoteRepository:
     extra_test_args = []
 
     class RPCError(Exception):
-        def __init__(self, name):
+        def __init__(self, name, rootCause):
             self.name = name
+            self.rootCause = rootCause
```

Comment:
pep8 names please, ie. `root_cause` (or maybe something more descriptive? How about `remote_[exception_]type` or something like that?